### ISSUE TITLE
Clarification on the debug.enableAllHovers setting

### ISSUE THREAD

**Issue by andrecasal** [microsoft/vscode-docs#3242]
I'd love some clarification on the debug.enableAllHovers setting.

From what @gregvanl said on [this issue](https://github.com/microsoft/vscode-docs/issues/3148#issuecomment-546417654), this setting is pertinent while [Live Sharing](https://visualstudio.microsoft.com/services/live-share/) a debug session and it has to do with how hovers are displayed on the Live Share client.

I've started a Live Share and debug session between VS Code and VS Code Insiders, but changing the debug.enableAllHovers to true or false doesn't have any effect. In fact, when I hover over pieces of code on the Live Share server, no hovers are shown on the Live Share client.

Could you please provide more detail as to what, exactly, this debug.enableAllHovers setting does?

**Comment by isidorn** [comment]
Hey @andrecasal I introduced this setting.
And from my memory this is only used by the Live Share client.
My suggestion is to simply not clarify that setting since it is used in an obscure way and should not be interesting to the users at all.

However I am not sure if the LiveShare ever picked up that setting.
Since the setting was introduced as a workaround , they might have found a diferent solution for an issue they were hitting.
Sorry for the vagues descirption, this was added 2 years ago

Bottom line, the setting will probably get deprecatd and we will remove it soon if Live Share does not use it. You can ask on the live share github repo for more details if they use it. Thanks

**Comment by andrecasal** [comment]
Hey @isidorn thanks for the info. I've [asked the question](https://github.com/MicrosoftDocs/live-share/issues/3002) on their repo.

**Comment by isidorn** [comment]
Great, let me know if it is not used by them and I can remove it.

**Comment by gregvanl** [comment]
@andrecasal Thanks for keeping us honest! ;-)

**Comment by andrecasal** [comment]
Haha you're welcome @gregvanl 😄
